src/views/v2/widgets/events-list.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Widget: Events List
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var array    $events             The array containing the events.
+ * @var string   $rest_url           The REST URL.
+ * @var string   $rest_nonce         The REST nonce.
+ * @var int      $should_manage_url  int containing if it should manage the URL.
+ * @var string[] $container_classes  Classes used for the container of the view.
+ * @var array    $container_data     An additional set of container `data` attributes.
+ * @var string   $breakpoint_pointer String we use as pointer to the current view we are setting up with breakpoints.
+ * @var array    $messages           An array of user-facing messages, managed by the View.
+ */
+
+$events = [
+
```

Comment:
* :no_entry_sign: Empty array declaration must have no space between the parentheses
* :no_entry_sign: Empty array declaration must have no space between the parentheses

<sub>hash: 3b761c1ddc756a1c46cde6fca7b7290e</sub>

Reply:
this is fixed in a later PR.